docs/descriptor/image.rst

```diff
@@ -8,6 +8,7 @@ Image descriptor contains all information Cekit needs to build and test a contai
 .. include:: version.rst
 .. include:: description.rst
 .. include:: from.rst
+.. include:: help.rst
```

Comment:
please include this into overrides/module descriptor pages too...

Reply:
Sorry can you give me an example of how you would like to see this incorporated?